test/interpreter/function-gcd.spec.ts

```diff
@@ -50,7 +50,7 @@ describe('Function GCD', () => {
   it('coerces to number', () => {
     const engine = HyperFormula.buildFromArray([
       ['=GCD("2",4)'],
-      ['=GCD(B2:C2)', '\'2', 4],
+      ['=GCD(B2:C2)', '2', 4],
```

Comment:
Why a change here? MULTINOMIAL in Excel will return `2` correctly when B2 is `'2`